app/models/bar.rb

```diff
@@ -16,4 +16,35 @@ def upvotes_score
   def downvotes_score
     (self.get_downvotes.size) * -1
   end
+
+  def self.popular_times(bars)
+    n = 0
+    bars.each do |bar|
+      @bar_name = bar.name
+      @bar_name += " #{bar.location}"
+      @bar_name.gsub(" ","+")
+      url = "http://www.google.com/search?q="+"#{@bar_name}"+"&num=10"
```

Comment:
Surrounding space missing for operator `+`.<br>Prefer `to_s` over string interpolation.